sdk/cosmos/azure_data_cosmos/docs/control-plane-warning.md

```diff
@@ -0,0 +1,5 @@
+<div class="warning">
```

Comment:
Are divs part of markdown?

Reply:
Markdown generally allows any HTML as well. More importantly, this is part of the rustdoc set of supported markdown: https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#adding-a-warning-block